### PR TITLE
[ADD] Update mutations & Visual feedback

### PR DESCRIPTION
**Pull request by Hai5edfm**
#15 
This PR adds the mutations working with real data and giving visual feedback to the users about the request status.
Also adds the verifications in the User update ( even if users do not add a field not required to 'update' it let the users to update just the fields they want)

### CODE REVIEW

**Review by AreYouIvan — APPROVED** [review]
Good job! 😎

File: src/components/ProductionMemberModal/index.tsx [hunk 1]
```diff
@@ -50,9 +57,14 @@ function ProductionMemberModal({
...
       console.log(res);
-      res.data?.errors
-      ? setReqStatus({error:{ errorMessage: res.data.errors[0].message}})
-      : setReqStatus({success: "Movie created successfully :D"})
+      if(reqStatus !== null) {
+        return;
+      } else {
+        res.data?.errors
+          ? setReqStatus({error:{ errorMessage: res.data.errors[0].message}})
+          : setReqStatus({success: "producer updated successfully :D"})
```
Comment (LilianaRestrepoTorres) [inline src/components/ProductionMemberModal/index.tsx:65]: Should it be like "{success:`${itemToUpdate} updated successfully`});"? @Hai5edfm

File: src/components/ProfileModal/index.tsx [hunk 2]
```diff
@@ -51,9 +57,13 @@ function ProfileModal({ openShowModal, setOpenShowModal }: ModalProps) {
...
       console.log(res);
-      // res.data?.errors
-      // ? setReqStatus({error:{ errorMessage: res.data.errors[0].message}})
-      // : setReqStatus({success: "Movie created successfully :D"})
+      if(reqStatus !== null) {
+        return;
+      } else {
+        res.data.errors !== undefined 
+          ? setReqStatus({error: {errorMessage: res.data.errors[0].message}}) 
+          : setReqStatus({success: 'User updated successfully'});
```
Comment (LilianaRestrepoTorres) [inline src/components/ProfileModal/index.tsx:65]: Should it be something like "{success:`${itemToUpdate} updated successfully`});"? @Hai5edfm

**Review by LilianaRestrepoTorres — APPROVED** [review]
Well done!

**Pull request merged**